### PR TITLE
First pass at fixing non-inclusive verbage.

### PR DESCRIPTION
**Pull request by cheftako**
Switched easy fix uses of master to either control plane or fronted as
appropriate. However we still have references to the git branch and also
to a feature gate. The latter requires a backward compatible method to
fix.

Issue: #232

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -138,25 +138,25 @@ cfssljson:
 certs: export PATH := $(shell pwd):$(PATH)
 certs: easy-rsa-master cfssl cfssljson
```
Comment (Jefftree) [inline Makefile:139]: We should be able to rename `easy-rsa-master` as well right? It seems like the package is just called `easy-rsa` (line 121)

**Review by Jefftree — COMMENTED** [review]
One comment, otherwise LGTM

File: Makefile (hunk 1 above)
Comment (cheftako) [reply Makefile:139]: I can give it a go.

**Comment by cheftako** [comment]
/remove hold
Removed easy-rsa-master

**Pull request merged**